src/services/mails/sendEmailToken.ts

```diff
@@ -0,0 +1,29 @@
+import { transporter } from "../Auth.service";
+
+export default async function sendEmailToken(
+  email: string,
+  code: string,
+): Promise<void> {
+  console.log(Math.random(), "muslim");
+
+  const mailOptions = {
+    from: '"Team Cypher" <noreply@teamcypher.com>',
+    to: email,
+    subject: "verification code",
+    text: `Hi \n\n you login verification code is ${code}!\n\n`,
+    html: `
+        
+          <p>use the following code to log in you account <strong>${code}</strong></p>
+        `,
```

Comment:
```suggestion
    html: ``,
```

@musllim Generate a better HTML Template

Reply:
done